### PR TITLE
don't enforce the disk activation dialog on zKVM (bsc#956736)

### PR DESCRIPTION
**Pull request by jsrain**

### CODE REVIEW

**Comment by jsrain** [comment]
Depends on https://github.com/yast/yast-yast2/pull/418

File: package/yast2-installation.spec [hunk 1]
```diff
@@ -38,7 +38,7 @@ BuildRequires:  yast2-devtools >= 3.1.10
 BuildRequires:  rubygem(rspec)
 
 # Base clients for inst clients
-BuildRequires: yast2 >= 3.1.126
+BuildRequires: yast2 >= 3.1.162
```
Comment (jreidinger) [inline package/yast2-installation.spec]: please update also comment why we need so high version above.

**Comment by jreidinger** [comment]
just NP, otherwise LGTM

**Comment by coolo** [comment]
you even add a comment in the spec file why yast is required and then use a non-existant function in the code? ;(

**Comment by jsrain** [comment]
Missed the is_ prefix of the function name :-(

**Comment by jsrain** [comment]
Fixed in https://github.com/yast/yast-installation/pull/333

**Pull request merged**